torch/csrc/distributed/rpc/RpcAgent.h

```diff
@@ -7,58 +7,66 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+using worker_id_t = uint64_t;
```

Comment:
Was wondering if we should make this int16_t similar to the distributed autograd context PR? Do we expect to have more than 65k workers? I'm assuming ranks will always be contiguous integers starting from 0. Is this assumption fair?

Reply:
Assumption makes sense. If we truncate ranks, I think we should also apply explicit checks, as we break promises implied by the `ProcessGroup` API.

Reply:
Is this `worker_id` part of our model parallel API and not really the `ProcessGroup` API? I feel we can truncate and add checks here to enforce this.